src/models/service/transactions/details.rs

```diff
@@ -155,41 +155,43 @@ pub struct MultisigExecutionDetails {
     pub safe_tx_gas: usize,
     pub base_gas: usize,
     pub gas_price: String,
+    // As this is a token we will keep the token information separated
```

Comment:
Do you want to open an issue for this? We could map this into `TokenInfo`. Another surprise breaking change.

![image](https://user-images.githubusercontent.com/11056031/125909776-b67a019d-2a4a-4386-8c5c-a3bd476af0ba.png)

Reply:
No, this was more a comment to explain what is going on. Some of our code is not always self explanatory :P